gitea/repositories/mock-turtle/examples/09 - Eelectron-quick-start/package-lock.json

```diff
@@ -439,7 +439,12 @@
       "integrity": "sha512-j5EctnkH7amfV/q5Hgmoal1g2QHFJRraOtmx0JpIqkxhBhI/lJSl1nMpQ45hVarwNETOoWEimndZ4QK0RHxuxQ==",
       "dev": true
     },
-
+    "minimist": {
```

Comment:
:exclamation:**Cycode**: Security vulnerability found in newly introduced dependency.
Dependency: minimist
Ecosystem: NPM
Identifiers: CVE-2021-44906
Severity: Critical
Fixed in version: 1.2.6
Dependency Paths: electron 7.2.4 -> extract-zip 1.7.0 -> mkdirp 0.5.5 -> minimist 1.2.5
Direct Dependency: No
Development Dependency: Yes
Vulnerability Description: Minimist <=1.2.5 is vulnerable to Prototype Pollution via file index.js, function setKey() (lines 69-95).


Would you like to exclude this security vulnerability from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_ignore_package_here** | Applies to this manifest for this package in this request only |
| **#cycode_ignore_manifest_here** | Applies to this manifest in this request only |
| **#cycode_ignore_package_everywhere** | Applies to this manifest for this package for all requests in your repository |

 <!-- 061c5d5b-2455-465f-afa1-7fa4d893178f -->